### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by sonyco-4u**
**This PR includes the following features:**
live demo = https://euphonious-crumble-fca2d2.netlify.app/

- Setup

1. Created react app.
2. Install dependencies.

- Basic setup

1. Created Header component.
2. Created MyProfile component.

- Created Rockets component and Redux state.

1. Created Rockets component.
2. Created Redux actions, state, reducers.
3. Fetched rockets details from SpaceX API.
4. Showed details of the rockets on the component.
5. Implemented Reserve/Cancel booking for rockets.

- Created Missions component and Redux state.

1. Created Missions component.
2. Created Redux actions, state, reducers.
3. Fetched missions details from SpaceX API.
4. Showed details of the missions on the component.
5. Implemented Reserve/Cancel booking for missions.

- Updated MyProfile Component

1. Implemented functionality to show reserved rockets on the component.
2. Implemented functionality to show reserved missions on the component.

- Applied styling for components

1. Applied styling for Header, MyProfile component.
2. Applied styling for Rockets component.
3. Applied styling for Missions component.

### CODE REVIEW

**Review by Lucas-Masaba — CHANGES_REQUESTED** [review]
Hello @sonyco-4u and @Gleennkar  👋 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request ⚠️ 

- [ ] Please add the missing headers to your README like that in the [default README template](https://github.com/microverseinc/readme-template). For example tech stack, key features, future features.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏 💻 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Lucas-Masaba in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by sonyco-4u** [comment]
@Lucas-Masaba , we have made the required corrections for the readme file, thanks.

File: src/components/missionsComponent/MissionItem.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+//       <td className="column name">{name}</td>
+//       <td className="column description">{description}</td>
+//       <td className="column status_column">
+//         {reserved
+//         && <p className="ActiveStatus">Active Member</p>}
+//         {!reserved
+//         && <p className="status">NOT A MEMBER</p>}
+//       </td>
+//       <td className="column column_button">
+//         {reserved
```
Comment (Kamba56) [inline src/components/missionsComponent/MissionItem.js:21]: - [Optional] Please remove all commented codes from your work. This is to keep your work clean.

**Review by Kamba56 — COMMENTED** [review]
Hi @Gleennkar @sonyco-4u , 

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 
🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉

Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**